node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -1866,7 +1871,14 @@ mod tests {
             .unwrap();
 
         let byte_set_to_compare = signed_transaction.raw_transaction.0;
-        assert_eq!(byte_set_to_compare.as_slice(), template)
+        eprintln!("signed {}", hex::encode(byte_set_to_compare.clone()));
```

Comment:
You'd like to remove the `eprintln!()` from here.

Reply:
is this resolved within this PR now @bertllll ?